src/content/app/tools/blast/components/blast-submission-header/BlastSubmissionHeader.scss

```diff
@@ -25,9 +25,9 @@
 }
 
 .showHide {
-  position: absolute;
   --show-hide-chevron-height: 8px;
   margin-left: 15px;
+  margin-right: -27px;
```

Comment:
Could you explain this? What was the problem with position: absolute for the ShowHide? It was sitting in the padding area if absolutely positioned.

![Screenshot 2022-11-15 at 13 44 19](https://user-images.githubusercontent.com/6834224/201934770-db6fc54e-8892-4b92-87a5-3cdb19e45883.png)

Reply:
with position: absolute, the chevron stayed where it was when scrolling the page.  
![image](https://user-images.githubusercontent.com/6040010/201939234-57dc72ed-0a84-490b-bb84-1a7ed29d7439.png)

Reply:
The right fix is by setting the parent position to relative. This has now been pushed.